Database Scripts/V021__insert_data_enemy_statistic_modifier.sql

```diff
@@ -0,0 +1,165 @@
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Haunted Hamburger'), 
+     -15),
+	 
+    ((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'ATTACK'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Ghoulish Grapes'), 
+     25), 
+    
+    ((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'DEFENSE'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Ghoulish Grapes'), 
+     15), 
+    
+    ((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'HEALTH'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Ghoulish Grapes'), 
+     -25),
+	 
+	((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'RIZZ'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Ghoulish Grapes'), 
+     -25),
+	 
+    ((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'ATTACK'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Sinister Sushi'), 
+     30), 
+    
+    ((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'DEFENSE'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Sinister Sushi'), 
+     20), 
+    
+    ((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'HEALTH'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Sinister Sushi'), 
+     -30),
+	 
+	((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'RIZZ'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Sinister Sushi'), 
+     -30),
+	 
+    ((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'ATTACK'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Menacing Muffin'), 
+     35), 
+    
+    ((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'DEFENSE'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Menacing Muffin'), 
+     25), 
+    
+    ((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'HEALTH'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Menacing Muffin'), 
+     -35),
+	 
+	((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'RIZZ'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Menacing Muffin'), 
+     -35),
+	 
+    ((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'ATTACK'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Wicked Waffle'), 
+     40), 
+    
+    ((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'DEFENSE'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Wicked Waffle'), 
+     30), 
+    
+    ((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'HEALTH'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Wicked Waffle'), 
+     -40),
+	
+	((SELECT [statisticTypeId] FROM [dbo].[StatisticType] WHERE [name] = 'RIZZ'), 
+     (SELECT [enemyId] FROM [dbo].[Enemy] WHERE [name] = 'Wicked Waffle'), 
```

Comment:
I believe it is Wayward Waffle instead of Wicked Waffle

Reply:
Changed to Wayward Waffle